policies/roadmap.md

```diff
@@ -0,0 +1,70 @@
+# OpenSSL project roadmap
+
+### Last modified 21 March 2022
+
+This document is the OpenSSL project roadmap. It is a living document
+and is expected to change over time. Objectives and dates should be
+considered aspirational.
+
+## Objectives
+
+Some of these objectives can be achieved more easily and quickly
+than others.
+
+#### Regular releases
+
+The OpenSSL project wants to move to having a regular release interval
+of approximately six months.
+
+#### QUIC
+
+OpenSSL is taking a multi-staged approach to the implementation of the
+[QUIC] transport protocol:
+
+- For OpenSSL 3.1, the focus is on providing a client side single stream
+QUIC implementation.
+
+- OpenSSL 3.2 will follow approximately six months later implementing more
+of the protocol.
+
+- OpenSSL 3.3 aims to complete the implementation a further six moths later.
+
+[QUIC]: https://datatracker.ietf.org/doc/html/rfc9000
```

Comment:
I suggest being more precise with what we refer to:

``` markdown
[QUIC-TRANSPORT]: https://datatracker.ietf.org/doc/html/rfc9000
[QUIC-TLS]: https://datatracker.ietf.org/doc/html/rfc9001
[QUIC-RECOVERY]: https://datatracker.ietf.org/doc/html/rfc9002
```

(the names are derived from the reference section on those RFCs)

Reply:
I'm not seeing the benefit for the extra complexity this introduces.  If you know about QUIC, none of these references are useful.  If you don't then having three links seems overkill.

I could reference RFC 8999 instead, it's simpler and more introductory.

Reply:
[shrug] ok, maybe I'm too picky then...